research/sentiment_analysis/sentiment_main.py

```diff
@@ -1,6 +1,14 @@
+"""Main function for the sentiment analysis model.
+
+The model makes use of concatenation of two CNN layers with
+different kernel sizes. See `sentiment_model.py`
+for more details about the models.
+"""
+
 import argparse
 from data import dataset
 import sentiment_model
+import tensorflow as tf
```

Comment:
Please fix import order-- should be python builtins, newline, thirdparty (including tf), newline, local lib.

Reply:
ack